src/styles/main.scss

```
@@ -14,7 +14,7 @@ $screen-smaller-than-mobile: 399px;
   --theme-primary: #5770be;
   --theme-primary-light: #6575ea;
   --theme-primary-lighter: #e2e5fb;
-  --light-grey: #c4c4c4;
+  --light-grey: #6b7787;
```

Une solution serait de créer une nouvelle couleur CSS et de l'appliquer au bouton désactivé :
```suggestion
  --light-grey: #6b7787;
  --medium-grey: #c4c4c4;
```